lib/api/fields/class.php

```diff
@@ -98,7 +98,7 @@ private function add() {
 	}
 
 	/**
-	 * Standadrize field to beans format.
+	 * Standardize field to beans format.
```

Comment:
This change is conflicting with the new Fields API merge that happened yesterday.  Go ahead and put it back to the original.